shotover-proxy/src/frame/cassandra.rs

```diff
@@ -378,6 +378,7 @@ impl CassandraFrame {
             u8::from(self.version) | u8::from(self.operation.to_direction());
 
         let mut flags = Flags::default();
+        flags.set(Flags::COMPRESSION, compression != Compression::None);
```

Comment:
The compression flag should be unmodified right?

Reply:
I think I'm misunderstanding this comment, what do you mean by unmodified? Here we are setting it using the message's codec state.

Reply:
yeah I was just mistaken, sorry about that.

One thing worth noting here is that we are actually disregarding whether the client had originally requested this specific message to be compressed or not and are actually compressing every single message that goes through.
But I'm not worried about that, I doubt the fine-grained control of compressing or not compressing messages was that useful in the first place. If we do hit performance issues for it we should look to optimize our compression code path first (theres a TODO about that below)